src/Language/es/Auth.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace CodeIgniter\Shield\Language\es;
```

Comment:
This file is already in `develop`.
Please don't include changes other than this PR.